### ISSUE TITLE
learn.svelte.dev crashing

### ISSUE THREAD

**Issue by JamesHertz** [sveltejs/learn.svelte.dev#195]
I was happily doing tutors on learn.svelte.dev util I go to https://learn.svelte.dev/tutorial/select-bindings and this error pop up:

Error [ENOENT]: ENOENT: no such file or directory, open '/home/learnsveltedev-uthj/node_modules/@sveltejs/kit/src/core/config/default-error.html'

<img width="1436" alt="Screenshot 2023-01-26 at 1 31 39 AM" src="https://user-images.githubusercontent.com/84348195/214735842-4b19e916-b66c-437c-89ac-f62711076dd5.png">

**Comment by dummdidumm** [comment]
I can't reproduce this. Is it reliably reproducible for you?

**Comment by JamesHertz** [comment]
Yes, it is. It's a matter of going to the tutorial right before it (https://learn.svelte.dev/tutorial/textarea-inputs), solve it, add something to the text field. Then click on "Next: Select bindings". And that's it.
